Index: Warenkorb/src/DateiHandler.java
===================================================================
diff --git a/Warenkorb/src/DateiHandler.java b/Warenkorb/src/DateiHandler.java
--- a/Warenkorb/src/DateiHandler.java	
+++ b/Warenkorb/src/DateiHandler.java	
@@ -12,29 +12,18 @@
 {
 	BufferedReader reader;
 	ArrayList<String> stringList;
+	File f;
+	Path p;
 	BufferedWriter writer;
 
 	public DateiHandler(File f)
 	{
-		stringList = new ArrayList<>();
-		try
-		{
-			reader = new BufferedReader(new FileReader(f));
-		} catch (FileNotFoundException e)
-		{
-			e.printStackTrace();
-		}
+		this.f = f;
 	}
 
 	public DateiHandler(Path p)
 	{
-		try
-		{
-			reader = Files.newBufferedReader(p);
-		} catch (IOException e)
-		{
-			e.printStackTrace();
-		}
+		this.p = p;
 	}
 
 	public ArrayList<String> lesen()
